src/components/JourneyPatterns/index.tsx

```diff
@@ -77,7 +77,7 @@ const JourneyPatterns = ({ journeyPatterns, onChange, children }: Props) => {
             })}
             ref={textFieldRef}
           />
-          <div>
+          <div className={'confirm-dialog-buttons'}>
```

Comment:
This change seems unrelated?

Reply:
This was helping with buttons in this dialog, just came along the way while making custom styles
<img width="692" alt="Screenshot 2024-06-14 at 14 16 36" src="https://github.com/entur/enki/assets/85493679/d9a008ac-9523-4e5f-8120-151af948d9a4">
<img width="692" alt="Screenshot 2024-06-14 at 14 16 44" src="https://github.com/entur/enki/assets/85493679/25ed8a0f-e03c-40c4-a440-d3352f74d27a">

Reply:
Ah nice, thanks for clarifying

Reply:
This is actually not an issue in the Entur styling, but it hepled in our case and it doesn't cause problems in the normal scenario (that class is used in other dialogs too)